test/unit/nft-api.test.ts

```diff
@@ -1203,6 +1204,47 @@ describe('NFT module', () => {
     });
   });
 
+  describe('computeRarity()', () => {
+    const contractAddress = '0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d';
+    const tokenId = '7495';
+    const nftRarityResult: NftAttributeRarity[] = [
+      {
+        value: 'Aquamarine',
+        trait_type: 'Background',
+        prevalence: 0.1266
+      },
+      {
+        value: 'Cyborg',
+        trait_type: 'Eyes',
+        prevalence: 0.0108
+      }
+    ];
+
+    beforeEach(() => {
+      mock.onGet().reply(200, nftRarityResult);
+    });
+
+    it('calls with the correct parameters', async () => {
+      await alchemy.nft.computeRarity(contractAddress, tokenId);
+
+      expect(mock.history.get.length).toEqual(1);
+      expect(mock.history.get[0].params).toHaveProperty(
+        'contractAddress',
+        contractAddress
+      );
+      expect(mock.history.get[0].params).toHaveProperty('tokenId', tokenId);
```

Comment:
Once you update `trait_type` to `traitType`, can you also add a test to check that the method converts the field properly?

Reply:
Sure, no problem.